lib/airbrake-ruby/payload_truncator.rb

```diff
@@ -109,11 +109,12 @@ def truncate_string(str)
     # @return [void]
     # @note This method mutates +str+ for speed
```

Comment:
Sure. Also, would you prefer keeping the bang in the method name since it still modifies the argument when it's not frozen?